File: PHP Excel Examples/CellsApiDemo/Row/getRow.php
<?php

use Spire\Cloud\Excel\Sdk\Api\CellsApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new CellsApi($configuration);

$name = "GetRow.xlsx";
$sheetName = "Sheet4";
$rowIndex = 2;
$folder = "input";
$storage = null;
$result = $apiInstance->getRow($name, $sheetName, $rowIndex, $folder, $storage);

File: Python Excel Examples/CellsApiDemo/row/getRow.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.cells_api import CellsApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.cells_api.CellsApi(configuration)
name = "GetRow_1.xlsx"
sheetName = "Sheet4"
row_index = 2
storage = ""
folder = "/ExcelDocument/"

result = api.get_row(name, sheet_name=sheetName, row_index=row_index, folder=folder, storage=storage)